addons/better-terrain/editor/Dock.gd

```diff
@@ -350,30 +365,56 @@ func perform_edit_terrain(index: int, name: String, color: Color, type: int, cat
 
 func _on_draw_pressed() -> void:
 	draw_button.button_pressed = true
+	line_button.button_pressed = false
+	rectangle_button.button_pressed = false
+	fill_button.button_pressed = false
+
+
+func _on_line_pressed() -> void:
+	draw_button.button_pressed = false
+	line_button.button_pressed = true
 	rectangle_button.button_pressed = false
 	fill_button.button_pressed = false
 
 
 func _on_rectangle_pressed() -> void:
 	draw_button.button_pressed = false
+	line_button.button_pressed = false
 	rectangle_button.button_pressed = true
 	fill_button.button_pressed = false
 
 
 func _on_fill_pressed() -> void:
 	draw_button.button_pressed = false
+	line_button.button_pressed = false
 	rectangle_button.button_pressed = false
 	fill_button.button_pressed = true
 
 
+func _on_replace_pressed() -> void:
```

Comment:
This function appears to be unused, it can be removed (and if connected to a signal, unconnected).